backend/src/data/repositories/interview/interview.repository.ts

```diff
@@ -31,9 +31,14 @@ class Interview {
     const interview = await this.#InterviewModel
       .query()
       .select()
-      .withGraphJoined('courseCategory')
-      .withGraphJoined('interviewee')
-      .withGraphJoined('interviewer')
+      .withGraphFetched('courseCategory')
+      .withGraphFetched('interviewee(withoutPassword).[userDetails]')
+      .withGraphFetched('interviewer(withoutPassword).[userDetails]')
```

Comment:
You may try the folowing approach, which helped me with joining more than 1 graph:
`.withGraphJoined('[courseCategory, interviewee, interviewer]')` - it is about basic join, but I think, you can take properties you need after join, or as in your case try something like that `.withGraphJoined('[courseCategory, interviewee(withoutPassword).[userDetails], interviewer(withoutPassword).[userDetails]]')`

Reply:
good point